output-data/alcohol-level-method.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "cjsCode": "A",
+    "description": "Blood",
+    "spiCode": "A"
+  },
+  {
+    "cjsCode": "B",
+    "description": "Breath",
+    "spiCode": "B"
+  },
+  {
+    "cjsCode": "C",
+    "description": "Amfetamine",
+    "spiCode": "C"
+  },
+  {
+    "cjsCode": "D",
+    "description": "Benzoylecgonine",
+    "spiCode": "D"
+  },
+  {
+    "cjsCode": "E",
+    "description": "Clonazepam",
+    "spiCode": "E"
+  },
+  {
+    "cjsCode": "F",
```

Comment:
Are we sure these are all `alcohol-level-method`s?

Reply:
Naming came directly from `Bichard`- ignoring

Reply:
we're just copying the existing data structure for now